### ISSUE TITLE
SCSS page overriding all pages 

### ISSUE THREAD

**Issue by LWatsonlm** [ga-wdi-exercises/project2#725]
the Welcome.scss is overriding all pages. My Welcome controller is separate - I assumed using the welcome.scss for styling, it would only style my welcome controller pages.

screenshot of welcome.scss
http://screencast.com/t/pam3GBGDVV

screenshot of post.scss
http://screencast.com/t/oh5X6Nbnme1F

thanks!

**Comment by nolds9** [comment]
@LWatsonlm what does your `application.css` look like?

**Comment by LWatsonlm** [comment]
heyo, nick!

it's empty. all my css files are empty 
http://screencast.com/t/8C7P7e4CNYLE

**Comment by nolds9** [comment]
So it seems like this issue stems from one of the tradeoffs we get for using Rails to serve assets for us....

Under the hood, Rails is combining all of `css`/`scss` files together, which it then loads on every page. If you do not want this behavior, you will need to either to adjust the specificity of your CSS, so that it only targets  elements that you know will only show up on 1 view. 

Or the longer route would be to tell Rails basically, not include the `welcome.scss` file in what's known as the Asset Pipeline, and then manually link to the stylesheet on the view you want, but honestly that is a journey that gets complex very fast. If really want to go down that route, this resource might be useful...http://guides.rubyonrails.org/asset_pipeline.html#coding-links-to-assets

**Comment by LWatsonlm** [comment]
I was worried about this. So it's pretty much impossible to have a welcome splash page in rails without going the longer route?

Do you have any suggestions on getting the entire page blue instead of just the the html here? 
http://screencast.com/t/ruMdDNTcJ1

**Comment by nolds9** [comment]
@LWatsonlm, not going to say impossible, it just requires getting in the weeds a little bit.
One option would be to make the "blue" element a container that takes has up the whole page

**Comment by LWatsonlm** [comment]
sorry could you ELI5? :)

**Comment by LWatsonlm** [comment]
ah like 

`background-color: #c0c0c0;
position:absolute; 
top:0;
left:0;
width:100%;
height:100%;
z-index:100;`

**Comment by nolds9** [comment]
yes!

**Comment by nolds9** [comment]
@LWatsonlm cool if I close this?
